apps/src/templates/studioHomepages/CourseBlocksGradeBands.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: pegasusContentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  }
+};
+
+const CourseBlocksGradeBands = React.createClass({
```

Comment:
Does that mean we should go back and update any components that currently use `React.createClass` in anticipation? I've used it all over the place 😬